content/providers/01-ai-sdk-providers/02-openai.mdx

```diff
@@ -303,6 +303,30 @@ You need to change Zod `.nullish()` and `.optional()` to `.nullable()`.
 
 </Note>
 
+#### Logprobs
+
+OpenAI provides logprobs information for completion/chat models.
+You can access it in the `providerMetadata` object.
+
+```ts highlight="11"
+import { openai } from '@ai-sdk/openai';
+import { generateText } from 'ai';
+
+const result = await generateText({
+  model: openai('gpt-4o'),
+  prompt: 'Write a vegetarian lasagna recipe for 4 people.',
+  providerOptions: {
+    openai: {
+      logprobs: true,
```

Comment:
maybe add a comment that this could also be a number. are the provider options documented?

Reply:
yep they're documented, will do 🙏